lib/tinycolor.dart

```diff
@@ -13,12 +13,11 @@ export 'color_extension.dart';
 
 class TinyColor {
   final Color originalColor;
-  late Color _color;
+  Color _color;
 
-  TinyColor(Color color) : this.originalColor = color {
-    this._color =
-        Color.fromARGB(color.alpha, color.red, color.green, color.blue);
-  }
+  TinyColor(Color color)
+      : this.originalColor = color,
+        _color = color.copy();
```

Comment:
I was actually thinking just
```
TinyColor(Color color)
      : this.originalColor = color,
      _color = Color.fromARGB(alpha, red, green, blue);
```
or is that bad practice somehow? :) the extension seems overkill, just for the internals of a constructor. Or do you believe it is generally useful? Personally I haven't had the need yet.

Reply:
It would actually be (reference color):
```dart
TinyColor(Color color)
      : this.originalColor = color,
      _color = Color.fromARGB(color.alpha, color.red, color.green, color.blue);
```
I don't think it is overkill, since the extension is private. 
I am used to creating extensions for these cases to simplify the constructor and improve formatting, but it is just a matter of taste. 
Both ways will produce the same end result, as of before, it was allowing the variable to not be initialized if somebody was to change the constructor and probably adding a tiny runtime penalty.

Reply:
yes, I forgot the color ref. 
sorry, I did not realize it was a private extension. thanks for explaining